hazelcast/src/main/java/com/hazelcast/spi/impl/NodeEngineImpl.java

```diff
@@ -426,10 +426,10 @@ public void reset() {
     public void shutdown(final boolean terminate) {
         logger.finest("Shutting down services...");
         waitNotifyService.shutdown();
+        operationService.shutdown();
```

Comment:
After this shift some enterprise-side tests start to throw time-out exceptions. https://hazelcast-l337.ci.cloudbees.com/job/new-lab-fast-EE-pr/954/console

An example cause is [here](https://github.com/hazelcast/hazelcast-enterprise/blob/master/hazelcast-enterprise/src/main/java/com/hazelcast/map/impl/EnterpriseMapServiceContextImpl.java#L257) an operation is added after operationService shutdown. 

@mdogan what can be done here?